custom-login/src/app/messages/messages.component.ts

```diff
@@ -17,8 +17,8 @@ import { HttpClient } from '@angular/common/http';
 import sampleConfig from '../app.config';
 
 interface Message {
-  date: String,
-  text: String,
+  date: String;
```

Comment:
String is lowercase `string` in TypeScript. https://www.typescriptlang.org/docs/handbook/basic-types.html#string